common/lib/dependabot/clients/bitbucket.rb

```diff
@@ -144,6 +151,23 @@ def create_pull_request(repo, pr_name, source_branch, target_branch,
       end
       # rubocop:enable Metrics/ParameterLists
 
+      def decline_pull_request(repo, pr_id, comment = nil)
+        # https://developer.atlassian.com/cloud/bitbucket/rest/api-group-pullrequests/
+        decline_path = "#{repo}/pullrequests/#{pr_id}/decline"
+        post(base_url + decline_path, "")
+
+        comment = "Dependabot declined pull request" if comment.nil?
```

Comment:
```suggestion
        comment = "Dependabot declined the pull request." if comment.nil?
```

Reply:
I applied the suggested change (also in the test).